Integration/cptIMSCobraExport_frm.frm

```diff
@@ -13,8 +13,6 @@ Attribute VB_GlobalNameSpace = False
 Attribute VB_Creatable = False
 Attribute VB_PredeclaredId = True
 Attribute VB_Exposed = False
-'<cpt_version>v3.1.3</cpt_version>

```

Comment:
Need this line for updates to recognize installed version vs. current version online. Since this is marked urgent, I'll add it.